engine/src/flutter/lib/web_ui/test/engine/app_bootstrap_test.dart

```diff
@@ -102,13 +103,12 @@ void testMain() {
       final Object appInitializer = await promiseToFuture<Object>(
         callMethod<Object>(engineInitializer, 'initializeEngine', <Object?>[]),
       );
-      final Object maybeApp = await promiseToFuture<Object>(
+      final FlutterApp maybeApp = await promiseToFuture<FlutterApp>(
         callMethod<Object>(appInitializer, 'runApp', <Object?>[]),
       );
 
-      expect(maybeApp, isA<FlutterApp>());
-      expect(getJsProperty<dynamic>(maybeApp, 'addView'), isA<Function>());
-      expect(getJsProperty<dynamic>(maybeApp, 'removeView'), isA<Function>());
+      expect(maybeApp['addView'], isA<Function>());
```

Comment:
I don't know how dart2wasm was passing this test before, maybe it's never run there?

I'd expect the expectation to fail because the result would be a `JSValue`, not a `Function`. I'd do something like `maybeApp['addView'].isA<JSFunction>, true` instead.

Reply:
ack